src/redux/actions/comments.js

```diff
@@ -0,0 +1,68 @@
+import axios from 'axios';
+import { POST_COMMENT, DELETE_COMMENT, EDIT_COMMENT } from './actionTypes';
+
+const apiUrl = 'https://vidar-ah-backend-production.herokuapp.com/api/v1';
+
+export const createComment = (slug, comments) => async (dispatch) => {
+  try {
+    const token = localStorage.getItem('token');
+
+    const res = await axios.post(`${apiUrl}/articles/${slug}/comments`, {
```

Comment:
@olajide1234 please name this appropriately instead of `res`

Reply:
Now updated as requested.